<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayAdjuster</class>
 <widget class="QWidget" name="DisplayAdjuster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="zoom_level">
     <property name="placeholderText">
      <string>Zoom Level (Int):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPlainTextEdit" name="start_time">
     <property name="placeholderText">
      <string>Start Time (Int):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>End Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="open_seperate_window">
     <property name="text">
      <string>Open Seperate Window</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Display Adjuster Window</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPlainTextEdit" name="end_time">
     <property name="placeholderText">
      <string>End Time (Int):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Start Time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Zoom Level</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
